docs/stories/Introduction.stories.mdx

```diff
@@ -1,207 +1,42 @@
-
-  @media all and (-ms-high-contrast:none) {
-  .link-list {
-      display: -ms-grid;
-      -ms-grid-columns: 1fr 1fr;
-      -ms-grid-rows: 1fr 1fr;
-    }
-  }
-
-  .link-item {
-    display: block;
-    padding: 20px 30px 20px 15px;
-    border: 1px solid #00000010;
-    border-radius: 5px;
-    transition: background 150ms ease-out, border 150ms ease-out, transform 150ms ease-out;
-    color: #333333;
+  .containt-logo {
     display: flex;
-    align-items: flex-start;
-  }
-
-  .link-item:hover {
-    border-color: #1EA7FD50;
-    transform: translate3d(0, -3px, 0);
-    box-shadow: rgba(0, 0, 0, 0.08) 0 3px 10px 0;
-  }
-
-  .link-item:active {
-    border-color: #1EA7FD;
-    transform: translate3d(0, 0, 0);
-  }
-
-  .link-item strong {
-    font-weight: 700;
-    display: block;
-    margin-bottom: 2px;
+    justify-content: center;
   }
-  
-  .link-item img {
-    height: 40px;
-    width: 40px;
-    margin-right: 15px;
-    flex: none;
-  }
-
-  .link-item span {
-    font-size: 14px;
-    line-height: 20px;
-  }
-
-  .tip {
-    display: inline-block;
-    border-radius: 1em;
-    font-size: 11px;
-    line-height: 12px;
-    font-weight: 700;
-    background: #E7FDD8;
-    color: #66BF3C;
-    padding: 4px 12px;
-    margin-right: 10px;
-    vertical-align: top;
+  .logo-logcomex {
+    width: 50vw;
   }
-
-  .tip-wrapper {
-    font-size: 13px;
-    line-height: 20px;
-    margin-top: 40px;
-    margin-bottom: 40px;
+  .containt-topics {
+    display: flex;
+    justify-content: center;
   }
-
-  .tip-wrapper code {
-    font-size: 12px;
-    display: inline-block;
+  .title-about{
+    text-align: center;
   }
-
-  
 `}</style>
 
-# Welcome to Storybook
+<!--About session-->
+<div className="title-about">
+  <h1>About our design system</h1>
+</div>
 
-Storybook helps you build UI components in isolation from your app's business logic, data, and context.
-That makes it easy to develop hard-to-reach states. Save these UI states as **stories** to revisit during development, testing, or QA.
+Ayla, [LogComex](https://www.logcomex.com/) Design System, was developed with the aim of facilitating the work of our team of developers.
+Combining design and agility, you can create new LogComex products without wasting time and following the pattern of our other products!
 
-Browse example stories now by navigating to them in the sidebar.
-View their code in the `src/stories` directory to learn how they work.
-We recommend building UIs with a [**component-driven**](https://componentdriven.org) process starting with atomic components and ending with pages.
+The Design System - Ayla, from LogComex, can be understood as an extension of [Vuetify](https://vuetifyjs.com/) components adapted to the needs of our team,
+also containing rules of layout, colors and typography of Log That way, we have all of our component standards here so that the team can enjoy them without the need to think about design rules and component functionalities, right?
```

Comment:
```suggestion
also containing rules of layout, colors and typography, we have all of our component standards here so that the team can enjoy them without the need to think about design rules and component functionalities.
```